source/frontend/src/features/mapSideBar/property/MotiInventoryHeader.tsx

```diff
@@ -38,7 +38,7 @@ export const MotiInventoryHeader: React.FunctionComponent<IMotiInventoryHeaderPr
           <Row className="no-gutters">
             <Col xs="8">
               <HeaderField label="Civic Address:" labelWidth={'3'} contentWidth="9">
-                {apiProperty?.address !== undefined ? formatApiAddress(apiProperty?.address) : '-'}
+                {exists(apiProperty?.address) ? formatApiAddress(apiProperty!.address) : '-'}
```

Comment:
this one probably needs your todo comment.

Reply:
I think we can do a search for ".!" and match places where we are ignoring the compiler